packages/react-slider/src/components/Slider/Slider.base.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
-import { FocusRects } from '../../Utilities';
+import { FocusRects } from '@uifabric/utilities';
```

Comment:
You could import from `office-ui-fabric-react/lib/Utilities` across the package to remove the dependency from `@uifabric/utilities` in the `package.json` file.

Reply:
I think importing from `@uifabric/utilities` is correct since this package actually should NOT depend on OUFR, and the OUFR dep will be removed once we figure out what to do with examples.

Reply:
yeah we want to avoid referencing OUFR as much as possible because that's temporary solution, see [this](https://hackmd.io/@fluentui/Hyu6U-XXP)